www_src/main.less

```diff
@@ -42,7 +42,7 @@ body {
   height: 100%;
   margin: 0;
   padding: 0;
-  font-family: 'Roboto', sans-serif;
+  font-family: "FiraSans", "Fira Sans", "Fira Sans OT", sans-serif;
```

Comment:
why "FiraSans" (is that the name it has when system-installed)? Also for pure [a-zA-Z]+ strings, CSS doesn't need quote symbols. Ironically that means that "sans-serif" *would* needs quotes around it except it's a baked in value, and using quotes around it would entirely the wrong thing (fun times!)